mobile_verifier/src/reward_shares.rs

```diff
@@ -109,6 +110,13 @@ impl TransferRewards {
     }
 }
 
+const BYTES_PER_DC: u64 = 20_000;
```

Comment:
it feels dangerous to have the same constant defined in multiple modules. Perhaps extract to a common `constants.rs`?

Reply:
the constant being in this crate is temporary and should removed tomorrow.